### PR TITLE
Adds some non-player references

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

Adds some references to people who don't play the game

## Why It's Good For The Game

It's funny I think.

## Changelog

:cl: Melbert
add:  Non-player references
/:cl:

### CODE REVIEW

**Comment by DaydreamIQ** [comment]
These truly are confusing times

File: strings/names/posibrain.txt [hunk 1]
```diff
@@ -21,29 +25,38 @@ HG
...
 MSO
+NOVA
 ORNG
 OSI
 PBU
 PHL
 PKP
 RR
 RZH
+SANS
```
Comment (Zonespace27) [inline strings/names/posibrain.txt:50]: image

Comment (Overuseddart) [reply strings/names/posibrain.txt:50]: Definitely not San1984

**Comment by Wallemations** [comment]
How much do you need to not play to be commemorated for not doing so

File: strings/names/posibrain.txt (hunk 1 above)
Comment (san7890) [reply strings/names/posibrain.txt:50]: yea because i only have one "s" in my name

**Pull request merged**